src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -150,6 +153,13 @@ class LicenceService(
       conditionSubmissionStatus = conditionSubmissionStatus,
     )
 
+    is HdcVariationLicence -> toHdcVariation(
+      licence = licence,
+      earliestReleaseDate = earliestReleaseDate,
+      isEligibleForEarlyRelease = isEligibleForEarlyRelease,
+      conditionSubmissionStatus = conditionSubmissionStatus,
+    )
```

Comment:
This file seems to contain a lot of changes that are covered by other tickets (and not related to the creation endpoint) which there's no test coverage for?

Reply:
This was something I mentioned in standup, though it may have been a day that you were on leave.
I was slightly confused about the scope of this ticket vs the "endpoints" ticket. Whether this was purely create-only, meaning the UI journey would be entirely inaccessible (as `HdcVariationLicence` would be completely unhandled except for in the `createVariation` endpoint), or whether this should be the functional user-journey of creating an HDC variation. Eg create -> answer SPO/VLO questions -> Submit

I went with the latter, with the view that this work would be next to untestable via the UI and would make the endpoint work significantly larger without these additional bits. If you'd rather them be in separate PRs, I can break them down though.

Reply:
As for testing, you're right, I completely missed them somehow. I'll get some in.